contrib/devtools/symbol-check.py

```diff
@@ -15,15 +15,15 @@
 
 import lief #type:ignore
 
-# Debian 9 (Stretch) EOL: 2022. https://wiki.debian.org/DebianReleases#Production_Releases
+# Debian 10 (Buster) EOL: 2024. https://wiki.debian.org/LTS
 #
-# - g++ version 6.3.0 (https://packages.debian.org/search?suite=stretch&arch=any&searchon=names&keywords=g%2B%2B)
-# - libc version 2.24 (https://packages.debian.org/search?suite=stretch&arch=any&searchon=names&keywords=libc6)
+# - g++ version 8.3.0 (https://packages.debian.org/search?suite=stretch&arch=any&searchon=names&keywords=g%2B%2B)
```

Comment:
```suggestion
# - g++ version 8.3.0 (https://packages.debian.org/search?suite=buster&arch=any&searchon=names&keywords=g%2B%2B)
```

Reply:
why is g++ mentioned here at all? Isn't this about glibc, a run-time dependency, whereas g++ is a compile time dependency?

Reply:
We also have a runtime dependency on libgcc.

Reply:
Maybe clarify that for non-build people? (Changed my first comment)

Reply:
Updated to mention libgcc instead.